src/app/public/modules/page-summary/page-summary-image.component.ts

```diff
@@ -1,5 +1,9 @@
 import { Component } from '@angular/core';
 
+/**
+ * Displays an image in the summary to identify a record
```

Comment:
@Blackbaud-ToddRoberts, are you good with getting rid of the current recommendation: `We recommend that you do not include images just for decorative purposes because they are likely to distract users and interfere with task completion.`? Or should we port this over to the Design tab?

How about getting rid of the current example: `As the example below demonstrates, you can use this component in conjunction with the sky-avatar component to allow users to manage and upload images.`